ingest/src/sources/GitHubDataSource.ts

```diff
@@ -1,9 +1,6 @@
 import { Document } from "langchain/document";
-import {
-  GithubRepoLoader,
-  GithubRepoLoaderParams,
-} from "langchain/document_loaders/web/github";
-import { Page, logger } from "chat-core";
+import { GithubRepoLoaderParams } from "langchain/document_loaders/web/github";
```

Comment:
(not in this PR) we should refactor this to just use the Git data source

Reply:
We did!

Reply:
What we should do is delete the GitHub thing entirely.